<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Rubesh" ignored="0" name="Default test" tests="6" failures="1" timestamp="2020-06-26T20:39:30 IST" time="1.864" errors="0">
  <testcase name="runScenario" time="0.319" classname="com.rubesh.Runner.CustomTestNGRunner"/>
  <testcase name="runScenario" time="0.028" classname="com.rubesh.Runner.CustomTestNGRunner"/>
  <testcase name="runScenario" time="0.035" classname="com.rubesh.Runner.CustomTestNGRunner"/>
  <testcase name="runScenario" time="0.022" classname="com.rubesh.Runner.CustomTestNGRunner"/>
  <testcase name="runScenario" time="0.062" classname="com.rubesh.Runner.CustomTestNGRunner">
    <failure type="java.lang.AssertionError" message="expected [The Lion, the Witch and the Wardrobe] but found [The Devil in the White City]">
      <![CDATA[java.lang.AssertionError: expected [The Lion, the Witch and the Wardrobe] but found [The Devil in the White City]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.rubesh.Cucumber.App.i_check_for_the_in_step(App.java:93)
at âœ½.I check for the title in step(file:///D:/EclipseWorkSpace/Cucumber/src/main/java/feature/First.feature:45)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="0.03" classname="com.rubesh.Runner.CustomTestNGRunner"/>
</testsuite> <!-- Default test -->
